### ISSUE TITLE
[BUG]: aria-describedby not reading for buttons

### ISSUE THREAD

**Issue by tombrunet** [IBMa/equal-access#2538]
### Project

Screen Reader Simulator browser extension

### Browser

Chrome

### Description

See https://react.carbondesignsystem.com/?path=/story/components-codesnippet--inline

The simulator just reads: ["copy to clipboard", button] and doesn't read the code, which is in the aria-describedby. It should be ["copy to clipboard", button, "node -v"].
